.travis.yml

```diff
@@ -13,4 +13,6 @@ cache:
   directories:
     - node_modules
 
-script: yarn run jest --projects jest.*.config.js --maxWorkers=4 --reporters jest-silent-reporter
+script:
+  - yarn run jest --projects jest.*.config.js --maxWorkers=4 --reporters jest-silent-reporter
+  - yarn playground:build
```

Comment:
This might be useful to catch some possible problems with the packages.

As a follow up, we might also think about running some e2e tests...not a high prio though